171 feat i18n in application

Thechi2000 opened the pull request:
Adds internationalization for static content. Locale can be selected in the new Accessibility view, and is stored in the `news_settings` table (to be used later for the generation). Translations are stored in `lib/l10n`.

Currently only features English and French locales.

Also fixes a bug in the provider menu, causing it to close even when there were errors on platforms other than Android, and removes all debug prints.

Closes #171

pmdlt reviewed (commented):
Absolutely nothing to complain about, the code is clean and well structured. The application compiles well, just like before, but with the second language as well - top!

I've checked for any layout problems related to the French language, but it's all good! 

However, a few translations are missing, so I've put together a small, non-exhaustive selection. I invite a second reviewer to double-check!

- In the news view (the widget probably) there's a `Read More` link that isn't translated
- Fetch news button not translated
- Alarm view not translated

Alerts seem to have been broken too.

---

That's it, it's not blocking in itself so I didn't make any request changes, if you tell me it's ok/done in another PR then I'll approve without any problem.

pmdlt commented on lib/views/accessibility.dart:
```diff
@@ -0,0 +1,69 @@
+                    ),
+                    DropdownButton(
+                      value: loc.localeName,
+                      items: AppLocalizations.supportedLocales
```
I couldn't find the exact place where the names of the languages were defined, but `français` should be capitalized.

Thechi2000 replied:
It was handled by an external library, but I removed it to have more control over it.

pmdlt reviewed (dismissed):
Thanks for all the changes!

Thechi2000 commented:
Thanks for the comments ! I added the missing translations and fixed the locales' display name. About the alerts, is it related to this PR ? It is working fine on my Android.

quartztz reviewed (approved):
No comment to be made; flawless code all around ! Thank you for the feature, can be merged asap :D
